dvc/stage.py

```diff
@@ -120,6 +120,7 @@ class Stage(object):
     STAGE_FILE_SUFFIX = ".dvc"
 
     PARAM_MD5 = "md5"
+    PARAM_SHA256 = "sha256"
```

Comment:
how will we support additional checksums that were asked in the ticket? Adding them one by one like this is not scalable, it will be hard to read and extend it.